sql-cli/src/opensearch_sql_cli/main.py

```diff
@@ -71,6 +71,14 @@
     default="sql",
     help="SQL OR PPL",
 )
+@click.option(
+    "-t",
+    "--timeout",
+    "response_timeout",
+    type=click.INT,
+    default=10,
+    help="Timeout to await a response from the server"
```

Comment:
should the help message indicate it's in seconds? and is the default 10 seconds too short? I think frontend request times out 30 seconds by default

Reply:
10 sec is default timeout in PY client.

Reply:
Help message updated in dad0f8f86.